docs/ADR-31-signin-signup-kernel-website.md

```diff
@@ -0,0 +1,33 @@
+# ADR-31 - Signin/signup Kernel<>Website
+
+# Communication kernel<>website in the signin/signup flow
+
+## Key Files
+- Kernel
+  -`session/sagas.ts`
+- Website
+    - `src/components/auth/LoginContainer.tsx`
+    - `src/components/auth/EthLogin.tsx`
+    - `src/components/auth/wallet/WalletSelector.tsx`
+
+## Kernel <> Website
+Disclaimer: The whole document describes the environment after  this [PR](https://github.com/decentraland/explorer/pull/2453)
+
+The current implementation is based on events and data in the global store. Take this into account when reading the diagrams.
+An arrow from `kernel` to `websites` doesn't actually means a direct connection between the 2 projects.
+
+The following diagrams represents a high-level view of the system.
+- Sign-in
```

Comment:
Sign-in is appearing below the image, maybe we can use a header level (#) instead of a bullet point here.

Reply:
I like the idea!